as/NFT/assembly/index.ts

```diff
@@ -47,8 +47,16 @@ export function claim_media(tokenMetadata: TokenMetadata): Media {
     assert_deposit_attached(DESIGN_PRICE)
 
     /** Assert uniqueId is actually unique */
-
-    let design = new Media(tokenMetadata.media, tokenMetadata.extra)
+    let design = new Media(
+        tokenMetadata.title,
+        tokenMetadata.copies,
+        tokenMetadata.media,
```

Comment:
this is where we'll pass the static version (which will be deployed also to Arweave) so we need to go on front-end and adjust minting too, so that we are both deploying the code as it's happening now, plus the preview image, which I guess you need to find a way to generate based on the final version of the iFrame code. I hope that's clear.

Reply:
All clear!
I open this https://github.com/curaOS/source/issues/178